test/resources/CalendarExporterTest/testMultipleResourcesExport.OUT.1.iCal

```diff
@@ -0,0 +1,54 @@
+BEGIN:VCALENDAR
+VERSION:2.0
+PRODID:https://oerworldmap.org/
+CALSCALE:GREGORIAN
+METHOD:PUBLISH
+BEGIN:VEVENT
+SUMMARY:DigiLern 2012
+ORGANIZER;CN="Ottobrunn Grammar School":mailto:sekr@gymnasium-ottobrunn.de
+DTSTART:20120308
```

Comment:
Dates should be either http://icalendar.org/iCalendar-RFC-5545/3-3-5-date-time.html or http://icalendar.org/iCalendar-RFC-5545/3-3-4-date.html